boxsdk/client/client.py

```diff
@@ -524,6 +524,37 @@ def create_user(self, name, login=None, **user_attributes):
             response_object=response,
         )
 
+    def pending_collaborations(self, limit=None, offset=None, fields=None):
+        """
+        Get the entries in the pending collaborations using limit-offset paging.
+
+        :param limit:
+            The maximum number of entries to return per page. If not specified, then will use the server-side default.
+        :type limit:
+            `int` or None
+        :param offset:
+            The offset of the item at which to begin the response.
+        :type offset:
+            `int` or None
+        :param fields:
+            List of fields to request.
+        :type fields:
+            `Iterable` of `unicode`
+        :returns:
+            An iterator of the entries in the pending collaborations
+        :rtype:
+            :class:`BoxObjectCollection`
+        """
+        return LimitOffsetBasedObjectCollection(
+            session=self._session,
+            url=self.get_url('collaboration'),
```

Comment:
This URL doesn't look correct; I think it should be `self.get_url('collaborations')`.  On a related note, you're missing a test for this method.